client/src/components/ImageCarousel/ImageCarousel.test.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable no-undef */
```

Comment:
I found no neediness in these rules

Reply:
already removed /* eslint-disable no-undef */  after jest added to eslint environment.